components/server/src/auth/resource-access.spec.ts

```diff
@@ -782,6 +782,43 @@ class TestResourceAccess {
                 teamRole: "owner",
                 expectation: true,
             },
+            // prebuild workspace with repo access
+            {
+                name: "prebuild workspace get owner",
+                resourceKind: "workspace",
+                workspaceType: "prebuild",
+                isOwner: true,
+                teamRole: undefined,
+                repositoryAccess: true,
+                expectation: true,
+            },
+            {
+                name: "prebuild workspace get other",
+                resourceKind: "workspace",
+                workspaceType: "prebuild",
+                isOwner: false,
+                teamRole: undefined,
```

Comment:
What is this case where the user is not the owner but also doesn't have a team?

Reply:
They have read access to the repository (see one line :point_down: ).

That's the qualifier based on which we share access to prebuilds, and thus should be the same for workspaces (at least for those of type `prebuild`). :+1: 

This fixes the UI issue we see currently.

Reply:
Aha, makes sense.